src/tile-nina/tile-nina.scss

```diff
@@ -11,21 +12,10 @@
     top: 0;
   }
 
-  .tile-media {
-    position: absolute;
-    top: 0;
-    left: 0;
-    right: 0;
-    padding-bottom: 0;
-
-    img {
-      width: 100%;
-    }
-  }
-
   .tile-content {
-    margin-top: 49.25%;
-    margin-bottom: 7%;
+    padding-bottom: 99.99%;
+    @extend .polaris-bg-sky-1;
```

Comment:
You're using quite a lot of polaris-globals in this CSS - which we are planning to deprecate.
Some of which has already been deprecated.

Can we amend this to use the backgrounds and colors in Toolkit instead?

There's some more info here -  http://sky-uk.github.io/toolkit/#core-gradients

e.g.